doc/release-notes.md

```diff
@@ -167,8 +170,8 @@ dogecoin-cli setmaxconnections <desired_amount>
 ### Improve the traffic graph in the UI
 
 Significantly improves the graph in the UI's traffic monitor by not throwing
```

Comment:
The previous implementation didn't discard the data when the window was closed, it would discard the data when the time span changed.

Reply:
Fix added in #3047